mcs/class/corlib/Test/System/EnvironmentTest.cs

```diff
@@ -156,9 +156,13 @@ public void GetEnvironmentVariables_NewlySet ()
 		public void GetCommandLineArgs ()
 		{
 			string[] args = Environment.GetCommandLineArgs ();
+#if !__WATCHOS__
 			Assert.IsNotNull (args, "not null");
 			Assert.IsTrue (((args.Length > 0) && (args.Length < 256)), "reasonable");
 			Assert.IsNotNull (args [0], "application");
+#else
+			Assert.AreEqual (0, args.Length, "length");
+#endif
```

Comment:
I think we should just edit GetCommandLineArgs to return an empty collection instead so consumers don't need to deal with these differences.

We even have code in the BCL that relies on this not being null: https://github.com/mono/mono/blob/91734968b83a0971c041254286cdc903b26564e2/mcs/class/corlib/System/Environment.cs#L129